packages/core/src/App/Constants/routes-config.js

```diff
@@ -261,6 +267,12 @@ const getModules = () => {
             is_authenticated: true,
             getTitle: () => localize('Wallets'),
         },
+        {
+            path: routes.cashier_p2p_v2,
```

Comment:
```suggestion
            path: routes.p2p_v2,
```

Reply:
If its no longer under cashier route

Reply:
for now it will remain under cashier route